ui/src/main/picsureui/search-interface/visualization-modal-view.hbs

```diff
@@ -0,0 +1,15 @@
+<div id="visualizations-modal">
+    <h5>All visualizations display the distributions of each variable for the specified cohort of interest.</h5>
+    <h5>{{message}}</h5>
+    <div class="visualization-body">
+        <div class="center large" id="spinner-holder">{{#if spinning}}{{> spinner spinnerClasses}}{{/if}}</div>
+        {{#if errors}}
+            {{#unless spinning}}
+                <div class="no-visualizations">
+                    <p>No visualizations available</p>
+                </div>
+            {{/unless}}
```

Comment:
Should errors be displayed at all? Usually even if it's not desirable for errors to happen, users getting info that an error occurred is better than them guessing.

Reply:
This approach is consistent with the rest of our biodatacatalyst UIs.